helm/block-producer/templates/block-producer.yaml

```diff
@@ -330,6 +335,21 @@ spec:
             path: keyfile.json
       {{- end }}
 ---
+{{- if $.Values.coda.persistentStorage }}
```

Comment:
Hey nice! Can you add a variable for the PVC storage class? We have a few storage classes setup for persistence.